Testreglar/251/Nett/nett-2.5.1a.json

```diff
@@ -0,0 +1,127 @@
+{
+	"id": "nett-2.5.1a",
+	"type": "web",
+	"kravTilSamsvar": "<p>All funksjonalitet som krever at brukeren benytter flerpunkts- eller stibasert gest, skal kunne betjenes med en enkelt pekerbevegelse</p>\r\n<ul>\r\n<li>Med mindre flerpunkts- eller stibasert gest er avgjørende</li>\r\n</ul>",
```

Comment:
Der mangler den del af kravet som hedder "kan betjenes ... uten stibasert gest" her, samt i "expectation" i 3.4. Det er medtaget i "applicability" i 3.2 og 3.3.
Er dette meningen?

Reply:
Vi har forenklet setningen og første setning gir mening om "... uten stibasert gest". Jeg tror ikke om det er behov for å gjenta det samme som allerede står i første del av setningen.